custom_components/tahoma/cover.py

```diff
@@ -339,6 +342,17 @@ def is_closing(self):
             for execution in self.coordinator.executions.values()
         )
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        attr = super().device_state_attributes or {}
+
+        # Obstruction Detected attribute is used by HomeKit
+        if self.has_state(IO_PRIORITY_LOCK_LEVEL_STATE):
```

Comment:
TODO double check if this won't add this attribute to all covers which have this state present (but empty)